src/main/java/filter/expression/Qualifier.java

```diff
@@ -441,6 +443,15 @@ public static boolean isUpdatedQualifier(Qualifier q) {
         }
     }
 
+    public static boolean isCountQualifier(Qualifier q) {
+        switch (q.getType()) {
```

Comment:
This can just be `q.getType() == QualifierType.COUNT`. The use of switch is left over from when the type was a string and we had to account for aliases.